### PR TITLE
`emphasize-draft-pr-label` - Support new issue view

### PR DESCRIPTION
**Pull request by kovsu**
- For #7856 

## Test URLs

https://github.com/download-directory/download-directory.github.io/issues?q=is%3Apr

## Screenshot

image

image

### CODE REVIEW

File: source/features/emphasize-draft-pr-label.css [hunk 1]
```diff
@@ -2,12 +2,17 @@
...
-	[aria-label='Open draft pull request'],
-	/* Global PR lists */
-	[aria-label='Draft Pull Request']
-) svg {
+	[rgh-emphasize-draft-pr-label]
+		.js-issue-row
+		:is(
+			[aria-label='Open draft pull request'],
+			[aria-label='Draft Pull Request']
+		)
```
Comment (fregante) [inline source/features/emphasize-draft-pr-label.css:10]: Might be worth checking if these are still applied on those two pages.

Also make sure comments are not lost

Comment (kovsu) [reply source/features/emphasize-draft-pr-label.css:10]: > Might be worth checking if these are still applied on those two pages.

Yeah, it still needs these.

File: source/features/emphasize-draft-pr-label.css [hunk 2]
```diff
@@ -2,12 +2,17 @@
 
 /* stylelint-disable-next-line media-feature-name-no-vendor-prefix -- It's the only cross-browser media query */
 @media (-webkit-min-device-pixel-ratio: 2) {
-	[rgh-emphasize-draft-pr-label] .js-issue-row :is(
-	/* Repo PR lists */
-	[aria-label='Open draft pull request'],
-	/* Global PR lists */
-	[aria-label='Draft Pull Request']
```
Comment (fregante) [inline source/features/emphasize-draft-pr-label.css:9]: These two comments were lost

**Comment by darkred** [comment]
Sorry, but correct me if I'm wrong, I don't see any difference with the feature enabled (v24.11.5 on both Chrome and Firefox). Below is a screenshot comparison. Should I file an issue?


Test URL: https://github.com/download-directory/download-directory.github.io/issues?q=sort%3Aupdated-desc+is%3Apr
*I also tested in the RG repo where the new issues is not enabled, no difference.*

<details>
<summary>feature disabled:</summary>


</details>

<details>
<summary>feature enabled:</summary>


</details>

**Comment by kovsu** [comment]
🤔 It actually works on my end.

image

**Comment by fregante** [comment]
@darkred the feature only applies to high resolution screens, see source and blame

**Pull request merged**